Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "You know, I recently thought about water activities that we could do together.How about diving?"
+    m 3eub "I read many books about the underwater world, and I would like to see this beauty myself.Just imagine."
```

Comment:
I would put this in past tense.
```suggestion
    m 3eub "I've read many books about the underwater world, and I'd like to see this beauty for myself. Just imagine..."
```
I would also change the expression here to be a closed eyes one, to better represent Monika imagining the experience.